src/ui/pages/work/partusage/PartUsageDialog.js

```diff
@@ -92,28 +105,29 @@ function PartUsageDialog(props) {
         updateFormDataProperty(FORM.PART, transactionLines.partCode);
         updateFormDataProperty(FORM.PART_DESC, transactionLines.partDesc);
         updateFormDataProperty(FORM.TRANSACTION_QTY, transactionLines.transactionQty);
+        updateFormDataProperty(FORM.LOT, transactionLines.lot);
     };
 
     useEffect(() => {
         if (isDialogOpen) {
-            initNewPartUsage();
+            runUiBlockingFunction(initNewPartUsage);
         } else {
-            setUoM('');
-            resetFormTransactionLinesAndBinListStates();
+            resetFormTransactionLinesAndRelatedStates();
+            updateFormDataProperty(FORM.ACTIVITY, null);
+            updateFormDataProperty(FORM.STORE, null);
             resetErrorMessages();
         }
     }, [isDialogOpen]);
 
     const initNewPartUsage = async () => {
-        setLoading(true);
         try {
             // Fetch the part usage object
             const response = await WSWorkorders.getInitNewPartUsage(workorder);
-            setLoading(false);
-            setInitPartUsageWSData(response.body.data);
-            assignInitialFormState(response.body.data);
+            const data = response.body.data;
```

Comment:
I can call it something like `defaultTransactionData` or `issueReturnPartTransactionData`, any of those work? I just put it into a `data` variable to avoid the repetition of `response.body.data`.

Reply:
Or another alternative could simply be `transactionData`

Reply:
Going to go with `defaultTransactionData`